scripts/storage_cleaner.py

```diff
@@ -1211,12 +1229,22 @@ def _get_src_dest_pairs_for_copy(
         for wandb_path in set(checkpoint_to_wandb_path.values())
     ]
 
+    if config.entry is not None:
+        src_dest_pairs = [
+            src_dest_pair
+            for src_dest_pair in src_dest_pairs
+            if Path(src_dest_pair[0]).name == config.entry.rstrip("/")
```

Comment:
This seems a little bit brittle, what about:

```suggestion
            if str(Path(src_dest_pair[0])).endswith(str(Path(config.entry)))
```

Do we warn anywhere if this list comes up empty?

Reply:
https://github.com/allenai/OLMo/pull/494/commits/6f56700a78eec2667792541e8147b868d20a0419 I found `Path.match` and it seems more robust (it won't match `test123` when `config.entry` is `123`, for example).

https://github.com/allenai/OLMo/pull/494/commits/b9268411f28e8f9122e5bd452e946e9be250f90f Made empty list an error.